calico-cloud_versioned_docs/version-20-2/network-policy/policy-tiers/tiered-policy.mdx

```diff
@@ -64,11 +64,11 @@ The default tier is where:
 
 ## AdminNetworkPolicy tier:
 
-The AdminNetworkPolicy tier is where all [Kubernetes admin network policies](https://network-policy-api.sigs.k8s.io/reference/examples/) reside. It is automatically created during installation and has the order of 1,000.
+The AdminNetworkPolicy tier is where all [Kubernetes admin network policies](https://network-policy-api.sigs.k8s.io/reference/examples/) reside. It is automatically created during installation and has the order of 1,000. This value is fixed, and cannot be changed.
```

Comment:
@ctauchen Can you please apply the same change to Calico open source and enterprise. I think we should try to keep them in sync if possible.

Reply:
I think the only mistake was leaving this out of CC. I took this copy from the CE docs. Thanks!